### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by DanUshindi**
## Portfolio: mobile menu 
In this  project I :
-  Added the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

- In mobile, implement the following features:

  When the user clicks (or taps) the hamburger button, the mobile menu appears.
  When the user clicks (or taps) the close (X) button, the mobile menu disappears.
  When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
  When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,14 +19,15 @@
...
     <div class="logo">
       <a href="#" class="link" target=" blank">DU</a>
     </div>
 
-    <button type="button" class="hamburger">
+    <button type="button" class="hamburger" onclick="menuFunction()">
       <span class="bars"></span>
     </button>
+   
     <nav class="nav-bar">
```
Comment (Hamzaoutdoors) [inline index.html:31]: **Great job making your hamburger menu work as expected, but there are some issues that you still need to work on to go to the next project but you are almost there!** : 

- [ ]  The mobile menu should have margins as defined in the design. Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to make the hamburger menu closely similar to the [original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 😸.
- [ ] Kindly add the blur background images as you can see in the screenshots below
- [ ] Kindly make sure that the mobile menu has a `height: 100vh;` 

**Current mobile menu**  |  **Original design**
:----------------------------:|:-------------------------:
  |

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @DanUshindi, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ The navbar menu works as expected 

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @hamzaoutdoors in your question so I can receive the notification ^_^

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -48,6 +48,94 @@ body {
...
 
+#mobile-menu-container {
+  display: none;
+  background-color: #191765;
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  position: absolute;
+  z-index: 999;
+}
```
Comment (medaminedev66) [inline style.css:59]: - [ ] Kindly add background images to your Menu as requested by the previous reviewer.
- [ ] Kindly try to fix the position of the Menu by adding  `position: fixed`  so your Menu is not scrollable.

- Your design:
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @DanUshindi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by medaminedev66 — APPROVED** [review]
# Hi @DanUshindi , 

## Status: Approved ✔️

:+1: You have implemented all the suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can consider:
- [ ] After closing your menu, your page is not scrollable. I would highly recommend fixing this issue before merging your PR for a better user experience. 

**You may merge the changes after fixing the issue**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Amine Smahi_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**